### PR TITLE
[wordpress__components] Change `isDefault` to `isSecondary` because `isDefault` is deprecated in `Button`

### PR DESCRIPTION
**Pull request by sydev**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/WordPress/gutenberg/tree/master/packages/components/src/button

### CODE REVIEW

File: types/wordpress__components/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
-// Type definitions for @wordpress/components 8.5
+// Type definitions for @wordpress/components 9.0
```
Comment (sirreal) [inline types/wordpress__components/index.d.ts]: 👍 Looks like this was _deprecated_ in 9.0

https://unpkg.com/browse/@wordpress/components@9.0.0/src/button/index.js

File: types/wordpress__components/button/index.d.ts [hunk 2]
```diff
@@ -3,9 +3,9 @@ import { ComponentType, HTMLProps } from '@wordpress/element';
 declare namespace Button {
     interface BaseProps {
         /**
-         * Renders a default button style.
+         * Renders a secondary button style.
          */
-        isDefault?: boolean;
```
Comment (sirreal) [inline types/wordpress__components/button/index.d.ts]: Rather than replacing it, should we add and deprecated, like the package has done?

Comment (sydev) [reply types/wordpress__components/button/index.d.ts]: This is an excellent idea, I have pushed the corresponding commit,

**Review by sirreal — APPROVED** [review]
Looks good, thanks 👍

**Comment by sirreal** [comment]
If you rebase or retrigger the tests in some other way, I suspect they'll pass. I had the same issue in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/42858 and they're passing now.

**Review by rbuckton — CHANGES_REQUESTED** [review]
There are a lot of unrelated changes showing up in the files changed. Could you either rebase or merge the latest from master?

**Comment by sydev** [comment]
I just rebased from master, but there are still many unrelated file changes. What else can I do?

**Comment by andrewbranch** [comment]
I’m going to close this since so many people were accidentally pinged on this PR and it doesn’t look like there’s been any progress toward sorting it out. @sydev please open a new PR if you still want to make these changes.

**Pull request closed without merging**